src/test/java/ch/zhaw/robopatrol/ServerTest.java

```diff
@@ -33,6 +33,21 @@ public void test() throws IOException {
         Main.stop();
     }
 
+    @Test(timeout = 30000)
+    public void testMain() throws IOException {
+        //int port = findPort();
+        Main.main(new String[0]);
```

Comment:
If we are using the default port for the test, then we might as well delete the `test()` above.